dashboard/app/controllers/application_controller.rb

```diff
@@ -9,7 +9,7 @@ class ApplicationController < ActionController::Base
   include ApplicationHelper
 
   include SeamlessDatabasePool::ControllerFilter
```

Comment:
Why do we retain this usage of Seamless here?

Reply:
I attempted to have `MultipleDatabasesTransitionHelper::ControllerFilter` extend from it so we could just include the one, but ControllerFilter is doing some funky stuff internally with its inheritance and initialization logic and I couldn't get it to work. I'm sure it'd be possible, but in the interests of time I decided to leave it as-is, and will just remove this line as part of the other removal work.